sensors/generic-sensor-tests.js

```diff
@@ -242,6 +250,24 @@ function runGenericSensorTests(sensorType) {
     sensor.stop();
   }, `${sensorType.name}: sensor receives suspend / resume notifications when\
   cross-origin subframe is focused`);
+
+  if (spatialSensors.indexOf(sensorType.name) == -1) {
+    // The sensorType does not represent a spatial sensor.
+    return;
+  }
+
+  promise_test(async t => {
+    const sensor = new sensorType({referenceFrame: "screen"});
+    const sensorWatcher = new EventWatcher(t, sensor, ["reading", "error"]);
+    sensor.start();
+
+    await sensorWatcher.wait_for("reading");
+    //TODO No way to test sensor APIs
```

Comment:
Can you explain the detailed TODO thing in the annotation?

Reply:
Updated the annotation.